flake.lock

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
@scientiac Is the standard way of sharing this file? or rather setting not unstable version and putting it to the `.gitignore`?